projects/plugins/jetpack/modules/stats.php

```diff
@@ -897,43 +898,44 @@ function stats_admin_bar_head() {
 }
 
 /**
- * Stats AdminBar.
+ * Gets the image source of the given stats chart.
  *
- * @access public
- * @param mixed $wp_admin_bar WPAdminBar.
- * @return void
+ * @param string $chart Name of the chart.
+ * @param array  $args Extra list of argument to use in the image source.
+ * @return string An image source.
  */
-function stats_admin_bar_menu( &$wp_admin_bar ) {
-	$url = add_query_arg( 'page', 'stats', admin_url( 'admin.php' ) ); // no menu_page_url() blog-side.
+function stats_get_image_chart_src( $chart, $args = array() ) {
```

Comment:
Nitpick: Can you add the `public` visibility keyword.

Reply:
I thought that was only possible on classes. Is that allowed in a global function? I get the following error when I do that: 
<img width="610" alt="Screen Shot 2021-11-09 at 11 32 54" src="https://user-images.githubusercontent.com/1233880/140908467-e73772f3-77b0-4afe-8198-253c40a59d88.png">

Reply:
Oh my, scratch that! I assumed it's a method part of a class. 🤦